sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -313,11 +259,17 @@ def test_regression_synthetic():
 
 
 def test_feature_importances():
+    # smoke test to check that the gradient boosting expose an attribute
+    # feature_importances_
     X = np.array(X_reg, dtype=np.float32)
     y = np.array(y_reg, dtype=np.float32)
 
-    clf = GradientBoostingRegressor(n_estimators=100, max_depth=5,
-                                    min_samples_split=2, random_state=1)
+    clf = GradientBoostingRegressor(
+        n_estimators=100, max_depth=5, min_samples_split=2, random_state=1
+    )
```

Comment:
do we really need to set all these attributes for this test?

Reply:
He does not seem to me.